### PR TITLE
实现页面跳转

### PR DESCRIPTION
**Pull request by goddessting**

### CODE REVIEW

File: public/login.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default class Login extends Component {
                     <input id="name" name="name" type="text"/><br />
                     密 码：
                     <input id="password" name="password" type="password"  /><br />
-                    <Link to='/main'><input id="submit" type="submit" /></Link>
+                    <Link to='main'><input id="login" type="submit" value="登录"/></Link>
```
Comment (Aym-fuhong) [inline public/login.jsx]: /main

File: public/main.jsx [hunk 2]
```diff
@@ -3,6 +3,14 @@ import {Link} from 'react-router';
 
 export default class Main extends Component {
     render() {
-        return <div><Link to='mealInfo'>这里是主页面</Link></div>
+        return(
+            <div>
+                <Link to='login'>登录</Link>
```
Comment (Aym-fuhong) [inline public/main.jsx]: 缺少/

File: public/mealinfo.jsx [hunk 3]
```diff
@@ -3,7 +3,16 @@ import {Link} from 'react-router';
 
 export default class MealInfo extends Component {
     render() {
-        return <div><Link to='/order'>显示所点击的菜的信息</Link></div>
+        return (
+            <div>
+                <div id="busnissName">细面之家</div>
```
Comment (Aym-fuhong) [inline public/mealinfo.jsx]: delete id

File: public/entry.jsx [hunk 4]
```diff
@@ -36,8 +35,7 @@ ReactDOM.render(
 // use jquery
 console.log($('#content').text());
```
Comment (seafruit) [inline public/entry.jsx]: 删了

File: public/entry.jsx [hunk 5]
```diff
@@ -36,8 +35,7 @@ ReactDOM.render(
...
 
-// Notice!!!
```
Comment (seafruit) [inline public/entry.jsx:37]: 留着

**Comment by baixue1016** [comment]
:+1:

**Comment by seafruit** [comment]
：+1

**Comment by seafruit** [comment]
:+1:

**Pull request merged**